src/components/JsonLd.tsx

```
@@ -0,0 +1,74 @@
+    'inLanguage': locale,
+    'organizer': {
+      '@type': 'Organization',
+      'name': festivalName,
+      'url': baseUrl
+    }
+  };
+
+  // Use React.createElement instead of JSX to avoid potential issues with SSR
+  return React.createElement('script', {
+    type: 'application/ld+json',
+    dangerouslySetInnerHTML: {
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)